src/Raven.Server/Documents/Handlers/SubscriptionsHandler.cs

```diff
@@ -152,13 +152,27 @@ public async Task Create()
                 var disabled = GetBoolValueQueryString("disabled", required: false);
                 var mentor = options.MentorNode;
                 var subscriptionId = await Database.SubscriptionStorage.PutSubscription(options, id, disabled, mentor: mentor);
+
+                var name = options.Name ?? subscriptionId.ToString();
+
+                // need to wait on the relevant remote node
+                var node = Database.SubscriptionStorage.GetResponsibleNode(name);
```

Comment:
can we avoid opening 2 server-store transactions by adding ctx argument to this method and moving it next to `WaitForExecutionOnSpecificNode` ?

Reply:
Done